drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -204,7 +183,7 @@ def do_attempt(self, attempt, suffix=''):
                 # If this item has no input value set, we are done with it.
                 is_correct = True
                 feedback = item['feedback']['correct']
-            state = {'top': attempt['top'], 'left': attempt['left']}
+            state = {'top': attempt['top'], 'left': attempt['left'], 'position': 'absolute'}
```

Comment:
The `'position': 'absolute'` is just a flag for backwards compatibility, so that we know the stored `top` and `left` values are absolute. It might as well be called `'format': '2.0' or something like that.

Before this patch the values were relative. If we didn't have a flag and treated all stored values as absolute, the positions of values stored before this patch would be slightly off.

I hate the fact that we will have to keep a flag for backwards compatibility forever, but I can't think of a better way to do that.